packages/protoc-gen-connect-web/src/javascript.ts

```diff
@@ -26,7 +26,7 @@ export function generateJs(schema: Schema) {
     if (protoFile.services.length == 0) {
       continue;
     }
-    const file = schema.generateFile(protoFile.name + "_connectweb.js");
+    const file = schema.generateFile(protoFile.name + "_connect.js");
```

Comment:
Should not change the existing `protoc-gen-connect-web`.

Reply:
Ugh.  Sorry, was a find/replace mishap.